assets/aurora.tokenlist.json

```diff
@@ -436,6 +436,15 @@
         "ethereum"
       ]
     },
+    {
+      "chainId": 1313161554,
+      "address": "0xa33C3B53694419824722C10D99ad7cB16Ea62754",
+      "symbol": "MECHA",
+      "name": "Mecha Tracker",
+      "decimals": 18,
+      "logoURI": "https://raw.githubusercontent.com/aurora-is-near/bridge-assets/master/tokens/mecha.svg",
+      "tags": []
+    },
```

Comment:
Tokens list shouldn't be manually updated.

```suggestion
```

Reply:
i dont understand. is there a non-manually way to add the entry? or it isnt necessary for the pr

Reply:
Yes, it will be done later with a tokens list generation tool.